app/src/main/java/com/github/polybooks/activities/ListBooksActivity.kt

```diff
@@ -10,6 +10,9 @@ import com.github.polybooks.database.Query
 import com.github.polybooks.utils.setupNavbar
 import com.google.android.material.bottomnavigation.BottomNavigationView
 
+/**
+ * Activity to list Books
```

Comment:
The same comment applies for all 3 activities:
Maybe explicit the doc? If I'm reading a ListBookActivity, I don't need the doc to tell me that the activity will list books.
It'd be much more useful to read:
"This activity displays books. Each book is a rectangular horizontal item and it is possible to scroll vertically to see more books.

Reply:
I don't really agree with that. That's just discribing the UI. This is a class description